modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -0,0 +1,52 @@
+= Defining a Custom Ingress Configuration
+
+[NOTE]
+====
+The tasks outline in this topic must be performed by an IT administrator
+====
+
+Runtime Fabric on Self-managed Kubernetes enables you to edit a template file that specifies a custom Ingress configuration. Using this file you can define annotations, rewrite rules, the ingressClassName, and more.
+
+The template file is a Kubernetes ConfigMap file. After customizing the template, you can use the `rtfctl apply` command-line utility to apply the custom Ingress configuration to your Kubernetes cluster.
+
+If you do not apply a custom template file, Runtime Fabric uses a default configuration that works with the out-of-the-box Kubernetes Ingress configuration.
+
+== Procedure
+
+. Copy the following into a text file:
++
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ingress-config
+  namespace: rtf
+data:
+  ingress.annotations: |
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/ssl-redirect: true
```

Comment:
`nginx.ingress.kubernetes.io/ssl-redirect: true` I am open to remove this line specifically. As we got some feed back from field where they said some use case requires the  `ssl-redirect: false` Or rather leave it this to the default implementation of ingress controller being used.

Reply:
Agree

Reply:
This is the file I got from the spec. Please provide a revised version that contains exactly what we want the user modify.

Reply:
File is updated in the Google doc.